config/environments/test.rb

```diff
@@ -1,26 +1,28 @@
 require "active_support/core_ext/integer/time"
 
+# The test environment is used exclusively to run your application's
+# test suite. You never need to work with it otherwise. Remember that
+# your test database is "scratch space" for the test suite and is wiped
+# and recreated between test runs. Don't rely on the data there!
+
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # The test environment is used exclusively to run your application's
-  # test suite. You never need to work with it otherwise. Remember that
-  # your test database is "scratch space" for the test suite and is wiped
-  # and recreated between test runs. Don't rely on the data there!
+  # Turn false under Spring and add config.action_view.cache_template_loading = true.
   config.cache_classes = true
```

Comment:
```suggestion
  config.cache_classes = !Kernel.const_defined?(:Spring)
```

Reply:
If spring is defined, then we don't cache classes as explained in the comment. Just make it automatic. Yeah, no Spring. I think we had it a few years ago still. Not sure.

Reply:
Well, if we ever do use it again, I'd jsut change the config, as this comment instructs:
> # Turn false under Spring and add config.action_view.cache_template_loading = true.

Reply:
Using spring is a personal decision IMO. But whatever, not important.